install/depends

```diff
@@ -132,8 +142,7 @@ elif [ "${DISTRO}" = "fedora" ]; then
 #----------------------------------------------------------------------------------------------------------------------
 elif [ "${DISTRO}" = "gentoo" ]; then
 
-    # Nothing to install on gentoo as everything is included by default!
-    true
+    emerge --ask=n --quiet-build=y --noreplace app-misc/jq
```

Comment:
@marshall-mcmullen do we need to `emerge --sync` ?

Reply:
Copy what we did here in `install/recommends`

```
  173     portdir=$(portageq get_repo_path / gentoo)
  174     if [[ ! -d "${portdir}" ]]; then
  175         emerge-webrsync
  176     fi
  177
```